packages/core/src/components/table/table/test/expandable-row-autocollapse/auto-collapse.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+test.describe.serial('Expandable Table Rows', () => {
+  test('should collapse the second row when expanding the first row', async ({ page }) => {
+    // Navigate to the page containing the table
+    await page.goto('src/components/table/table/test/expandable-row-autocollapse/index.html');
+
+    // Select the row element by data-testid or row-id
+    const firstRowToggle = page.locator(
+      'tds-table-body-row-expandable[data-testid="1"] > tr > td > label',
+    );
+    // Expand the first row
+    await firstRowToggle.click();
+
+    // Assert that the first row is expanded
+    const firstRowContent = page.getByText(/Hello world 1/);
+    await expect(firstRowContent).toBeVisible();
+    /* check input screenshot diff */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('should collapse the first row when expanding the second row', async ({ page }) => {
+    // Navigate to the page containing the table
+    await page.goto('src/components/table/table/test/expandable-row-autocollapse/index.html');
```

Comment:
I think that it should be combined into one? Cos every page.goto gives a fresh state.

First test: Load page -> Click row 1 -> Check row 1
Second test: Load page (fresh) -> Click row 2 -> Check row 2
This means we never actually test what happens when row 1 is expanded AND THEN row 2 is clicked in the same session. The auto-collapse behavior (where expanding one row should collapse others) can only be tested when we perform these actions sequentially without a page reload in between.

Reply:
Yeah good catch!